packages/@okta/vuepress-site/docs/guides/request-user-consent/main/index.md

```diff
@@ -368,7 +395,7 @@ curl -v -X DELETE \
 "https://${yourOktaDomain}/api/v1/users/${userId}/grants"
 ```
 
-> **Note:** See [Revoke all Grants for a User](/docs/reference/api/users/#revoke-all-grants-for-a-user) for more information.
+> **Note:** See [Revoke all Grants for a User](/docs/reference/api/users/#revoke-all-grants-for-a-user).
 
```

Comment:
See Troubleshooting section below: remove the `/grants` endpoint from the URL link.